packages/detox/src/plugins/plugin.ts

```diff
@@ -91,10 +92,15 @@ function buildDetoxTargets(
   const targets: Record<string, TargetConfiguration> = {
     [options.buildTargetName]: {
       command: `detox build`,
+      dependsOn: [], // it does not depend on the build the app target
```

Comment:
If this is not set does it mean it will depend on the `buildTarget`?

Reply:
yes, actually setting dependsOn here does not do anything